fortran/src/H5Pff.F90

```diff
@@ -948,7 +948,7 @@ END SUBROUTINE h5pget_alignment_f
 !! \param backing_store Boolean flag indicating whether to write the file contents to disk when the file is closed.
 !! \param hdferr        \fortran_error
 !!
-!! See C API: @ref herr_t H5Pget_fapl_core(hid_t fapl_id, size_t *increment, hbool_t *backing_store);
+!! See C API: @ref H5Pget_fapl_core()
```

Comment:
should this be "See C API: @ref H5P**s**et_fapl_core()"? (SUBROUTINE is h5p**s**et_fapl_core_f, not **g**et).

Reply:
fixed.